backend/serializers/participant_serializer.py

```diff
@@ -4,10 +4,12 @@
 class ParticipantSchema(Schema):
     class Meta:
         fields = (
-            'id', 'name', 'lastname', 'email', 'github', 'hacknights', 'phone'
+            'id', 'participants_ids', 'name', 'lastname',
```

Comment:
I am not sure, if that's a good wat to do that with `participant_ids`.

Reply:
At first I delegated it to separate schema, but as we discussed, it wasn't best solution. So I pull it into ParticipantSchema. Do you have some suggestions. I certenly want serialize somehow that payload.

Reply:
@w1stler please solve this discussion and merge

Reply:
I could make it inline inside Resource:

```
ids_schema = Schema.from_dict({"participants_ids": fields.List(fields.Int())})
        try:
            data = ids_schema().load(json_data)
```

and remove from participant serializer.
What do you think about that solution?

Reply:
Seems that is a better solution. We can find some examples with good practices for similar problem, as I am still not 100% confident if that's the best approach. :)